src/renderer/containers/Validator/AddValidatorSigningKeysModal/index.tsx

```diff
@@ -54,12 +54,8 @@ const AddValidatorSigningKeysModal: FC<ComponentProps> = ({close}) => {
   };
 
   const checkPrivateSigningKey = (publicKey: string, privateKey: string): boolean => {
-    try {
-      const {accountNumberHex} = new Account(privateKey);
-      return accountNumberHex === publicKey;
-    } catch (error) {
-      return false;
-    }
+    const {accountNumberHex} = new Account(privateKey);
+    return Account.isValidPair(accountNumberHex, publicKey);
```

Comment:
You should be able to just do this:

```ts
const checkPrivateSigningKey = (publicKey: string, privateKey: string): boolean => 
  Account.isValidPair(privateKey, publicKey)
```

Note: the private key (signing key) goes second and the public key (account number) goes first.